src/enums.ts

```diff
@@ -93,12 +93,12 @@ export enum Condition {
 }
 
 export namespace Condition {
-  export function matchesBuilding(condition: Condition, building: Building, planet: Planet): boolean {
+  export function matchesBuilding(condition: Condition, building: Building, planet: Planet, faction: Faction): boolean {
     if (condition as string === building as string) {
       return true;
     }
     switch (condition) {
-      case Condition.MineOnGaia: return building === Building.Mine && planet === Planet.Gaia;
+      case Condition.MineOnGaia: return building === Building.Mine && planet === Planet.Gaia && faction !== Faction.Lantids;
```

Comment:
mines on other's planets are not getting any effect for gaia and pt. But you are right, this is impacting also regular mines. Need to correct this

Reply:
> it does not count for any effects that relate to how many planet types or Gaia planets you have colonized.

From what I understand, it does not count for how many planet types or how many gaia planets you have colonized. But it does count for when you build a new mine on a gaia planet, since it's not a "count" effect but a trigger effect.